src/Init/Notation.lean

```diff
@@ -476,6 +476,9 @@ instance : Coe Syntax (TSyntax `rawStx) where
 /-- `with_annotate_term stx e` annotates the lexical range of `stx : Syntax` with term info for `e`. -/
 scoped syntax (name := withAnnotateTerm) "with_annotate_term " rawStx ppSpace term : term
 
+/-- Term elaborator which uses the expected type to insert coercions. -/
```

Comment:
Is this intended for users? I think I would find it quite unhelpful in a docstring. Doesn't up-arrow already do this?

Reply:
OK, having read more of this PR I get what's going on a bit more. What about:

```suggestion
/-- Normalize casts in an expression using the same method as the `norm_cast` tactic. -/
```